### PR TITLE
Sidekiq worker

### PR DESCRIPTION
**Pull request by kayiwa**
need this to go in to work on bibdata role

### CODE REVIEW

**Comment by jrgriffiniii** [comment]
```
failed: E: Failed to fetch http://ppa.launchpad.net/brightbox/ruby-ng/ubuntu/pool/main/r/ruby2.4/ruby2.4-dev_2.4.6-1bbox1~bionic1_amd64.deb  Could not connect to ppa.launchpad.net:80 (91.189.95.83), connection timed out [IP: 91.189.95.83 80]
```
is triggering build failures in CircleCI containers.  I'll keep attempting to get the build passing.

**Comment by kayiwa** [comment]
Hey @jrgriffiniii this uses rvm now.

**Comment by kayiwa** [comment]
PING? okay to merge

File: roles/pulibrary.sidekiq_worker/molecule/default/tests/test_sidekiq_worker.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+def test_sidekiq_workers_systemd_service_file(host):
+    f = host.file('/etc/systemd/system/sidekiq-workers.service')
```
Comment (jrgriffiniii) [inline roles/pulibrary.sidekiq_worker/molecule/default/tests/test_sidekiq_worker.py]: 👍

**Pull request merged**